pkg/storage/utils/eosfs/eosfs.go

```diff
@@ -2138,6 +2118,27 @@ func (fs *Eosfs) getEOSToken(ctx context.Context, u *userpb.User, fn string) (eo
 	return eosclient.Authorization{Token: tkn}, nil
 }
 
+// // Returns an Authorization wich assumes the role of the owner of the file `fn`
+func (fs *Eosfs) getOwnerAuth(ctx context.Context, u *userpb.User, fn string) (eosclient.Authorization, error) {
```

Comment:
`user` is not used for anything, so it shouldn't be here (it would be clearer as well not to have it).  Removing it here also removes the need to get it from ctx.

But btw... Q: doesn't the metadata of the file (that you get before calling this) contain the owner?

Reply:
No. Because for example in `DownloadRevision` and `RestoreRevision`, a `Reference` is passed (so without any info)